### PR TITLE
Add stripe integration

### PR DESCRIPTION
**Pull request by sbauch**

### CODE REVIEW

File: src/pages/pricing/index.js [hunk 1]
```diff
@@ -29,76 +27,74 @@ function encode(data) {
...
-    titleStyles: {
-      borderRadius: "4px 4px 0 0 ",
-    },
-    markerColor: "#4E61A5",
   },
   {
     name: "Enterprise",
-    price: "Custom",
-    className: "enterpriseCard",
+    price: "from $10k/year",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:81]:
```suggestion
    price: "$10k+/year",
```

File: src/pages/pricing/styles.module.css [hunk 2]
```diff
@@ -19,72 +22,84 @@
...
 }
 
-.enterpriseCard .planHeader {
-  color: white;
+.price {
+  color: #4E61A5;
+}
+
+.planCopy {
+  height: 140px;
```
Comment (sccarmichael) [inline src/pages/pricing/styles.module.css:57]:
```suggestion
font-size: 16px;
    height: 140px;
```

File: src/pages/pricing/styles.module.css [hunk 3]
```diff
@@ -19,72 +22,84 @@
...
+
 .planFeature {
   font-size: 16px;
   color: #595959;
+  list-style-type: none;
+  padding-left: 32px;
+  margin-top: auto;
+}
+
+.planFeature:before {
```
Comment (sccarmichael) [inline src/pages/pricing/styles.module.css:78]: Not sure how to do this, but planCopy is running over planFeature when resizing -- i tried to tweak a few things but unsure best route.

Comment (sbauch) [reply src/pages/pricing/styles.module.css:79]: I had stuck a height on the `planCopy` - it felt weird to me when there was a shorter paragraph that the bullets on the bottom were not aligned.

I switched it to a min-height instead so should be good now

File: src/pages/pricing/index.js [hunk 4]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
 }
 
-const plans = [
+export const plans = [
   {
-    name: "Community",
-    price: "$0",
-    className: "",
+    name: "PMF",
+    price: "$49/mo",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:31]:
```suggestion
    price: "$99/mo",
```

File: src/pages/pricing/index.js [hunk 5]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
-      borderRadius: "4px 4px 0 0 ",
-    },
-    markerColor: "#4E61A5",
   },
   {
-    name: "Business",
-    price: "$249/mo",
-    className: "businessCard",
+    name: "Series A",
+    price: "$149/mo",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:47]:
```suggestion
    price: "$199/mo",
```

File: src/pages/pricing/index.js [hunk 6]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
 
-const plans = [
+export const plans = [
   {
-    name: "Community",
-    price: "$0",
-    className: "",
+    name: "PMF",
+    price: "$49/mo",
+    priceId: "osso_pmf",
```
Comment (sbauch) [inline src/pages/pricing/index.js:32]:
```suggestion
    priceId: "price_1HzqrZG25oQYthlcPgwp81JB",
```

File: src/pages/pricing/index.js [hunk 7]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
+      <span key="0">
+        Pick this if: you’ve got customer demand and have had SAML in your
+        backlog for too long
+      </span>,
+    ],
+  },
+  {
+    name: "Unicorn",
+    price: "$499/mo",
+    priceId: "osso_pmf", // TODO
```
Comment (sbauch) [inline src/pages/pricing/index.js:65]:
```suggestion
    priceId: "price_1Hzqv3G25oQYthlcnLuHacWO",
```

File: src/pages/pricing/index.js [hunk 8]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
-    },
-    markerColor: "#4E61A5",
   },
   {
-    name: "Business",
-    price: "$249/mo",
-    className: "businessCard",
+    name: "Series A",
+    price: "$149/mo",
+    priceId: "osso_pmf", // TODO
```
Comment (sbauch) [inline src/pages/pricing/index.js:48]:
```suggestion
    priceId: "price_1HzqunG25oQYthlcEv25U7OB",
```

File: functions/create-checkout.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    mode: "subscription",
+    line_items: [
+      {
+        price: params.priceId,
+        quantity: 1,
+      },
+    ],
+    subscription_data: {
+      trial_period_days: 21,
+    },
```
Comment (sbauch) [inline functions/create-checkout.js:20]:
```suggestion

```

File: src/pages/pricing/index.js [hunk 10]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
-const plans = [
+export const plans = [
   {
-    name: "Community",
-    price: "$0",
-    className: "",
+    name: "PMF",
+    price: "$99/mo",
+    priceId: "price_1HzqrZG25oQYthlcPgwp81JB",
     cta: "Start for free",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:33]:
```suggestion
    cta: "Choose PMF",
```

File: src/pages/pricing/index.js [hunk 11]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
-    markerColor: "#4E61A5",
   },
   {
-    name: "Business",
-    price: "$249/mo",
-    className: "businessCard",
+    name: "Series A",
+    price: "$199/mo",
+    priceId: "price_1HzqunG25oQYthlcEv25U7OB",
     cta: "Get started",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:49]:
```suggestion
    cta: "Choose Series A",
```

File: src/pages/pricing/index.js [hunk 12]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
+        Pick this if: you’ve got customer demand and have had SAML in your
+        backlog for too long
+      </span>,
+    ],
+  },
+  {
+    name: "Unicorn",
+    price: "$499/mo",
+    priceId: "price_1Hzqv3G25oQYthlcnLuHacWO",
+    cta: "Contact us",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:66]:
```suggestion
    cta: "Choose Unicorn",
```

File: src/pages/pricing/index.js [hunk 13]
```diff
@@ -27,78 +25,76 @@ function encode(data) {
...
-    },
-    markerColor: "#4E61A5",
   },
   {
     name: "Enterprise",
-    price: "Custom",
-    className: "enterpriseCard",
+    price: "$10k+/year",
+    priceId: "osso_pmf", // TODO
     cta: "Contact us",
```
Comment (sccarmichael) [inline src/pages/pricing/index.js:83]:
```suggestion
    cta: "Get in touch",
```

File: src/components/paymentModal.js [hunk 14]
```diff
@@ -0,0 +1,253 @@
+import { CardElement, useElements, useStripe } from "@stripe/react-stripe-js";
+import { Checkbox, Form, Input, Modal, Spin } from "antd";
+import React, { useState } from "react";
+
+const CARD_ELEMENT_OPTIONS = {
+  style: {
+    base: {
+      iconColor: "#4E61A5",
```
Comment (sccarmichael) [inline src/components/paymentModal.js:8]:
```suggestion
      iconColor: "#595959",
```

File: src/components/paymentModal.js [hunk 15]
```diff
@@ -0,0 +1,253 @@
...
+      color: "#bfbfbf",
```
Comment (sccarmichael) [inline src/components/paymentModal.js:9]:
```suggestion
      color: "#595959",
```

File: src/components/paymentModal.js [hunk 16]
```diff
@@ -0,0 +1,253 @@
...
+      fontWeight: 600,
```
Comment (sccarmichael) [inline src/components/paymentModal.js:10]:
```suggestion
      fontWeight: 400,
```

File: src/components/paymentModal/index.js [hunk 17]
```diff
@@ -0,0 +1,272 @@
...
+    });
+  };
+
+  return (
+    <Modal
+      visible={open}
+      title={`Checkout - Step ${customerId ? 2 : 1} of 2`}
+      width={640}
+      onCancel={onClose}
+      okText="Submit"
```
Comment (sccarmichael) [inline src/components/paymentModal/index.js:115]:
```suggestion
      okText="Next"
```

**Comment by sccarmichael** [comment]
One last small thing -- the `Back` button in step 2 still behaves as a `Cancel` and closes out the window. Can you set it to go back to the previous step?

**Review by sccarmichael — APPROVED** [review]
Leggo!

**Pull request merged**